packages/grafana-ui/src/components/Logs/LogMessageAnsi.tsx

```diff
@@ -12,6 +12,15 @@ interface ParsedChunk {
 
 function convertCSSToStyle(css: string): Style {
   return css.split(/;\s*/).reduce((accumulated, line) => {
+    // The ansicolor package returns this color if the chunk has the ANSI dim
+    // style (`\e[2m`), but it is nearly unreadable in the dark theme, so we use
+    // `opacity` instead to style it in a way that works across all themes.
+    if (line === 'color:rgba(0,0,0,0.5)') {
+      // @ts-ignore
+      accumulated['opacity'] = '0.5';
+      return accumulated;
+    }
+
```

Comment:
just a nit, I see this is already done in the same function so no biggie:

it would be great if instead of `@ts-ignore` which would suppress any error we could do
```suggestion
  return css.split(/;\s*/).reduce<Style>((accumulated, line) => {
    // The ansicolor package returns this color if the chunk has the ANSI dim
    // style (`\e[2m`), but it is nearly unreadable in the dark theme, so we use
    // `opacity` instead to style it in a way that works across all themes.
    if (line === 'color:rgba(0,0,0,0.5)') {
      accumulated['opacity'] = '0.5';
      return accumulated;
    }

```

if then you could also remove the ignore on line 28 that would be great :)

Reply:
This is much nicer, thank you 🙏